service/frontend/accessControlledHandler.go

```diff
@@ -73,6 +71,12 @@ func (a *AccessControlledWorkflowHandler) RegisterHandler() {
 	a.GetDispatcher().Register(metaserver.New(a))
 }
 
+// UpdateHealthStatus sets the health status for this rpc handler.
+// This health status will be used within the rpc health check handler
+func (a *AccessControlledWorkflowHandler) UpdateHealthStatus(status HealthStatus) {
+	a.frontendHandler.UpdateHealthStatus(status)
+}
+
 // Health callback for for health check
 func (a *AccessControlledWorkflowHandler) Health(ctx context.Context) (*health.HealthStatus, error) {
```

Comment:
Do we need to forward this call to the underlying frontendHandler?

Reply:
ofcourse, thanks for catching this.

Reply:
done